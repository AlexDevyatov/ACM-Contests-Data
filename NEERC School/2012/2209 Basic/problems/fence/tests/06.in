7
-29 -10
-26 -26
1 -29
5 -26
27 4
28 26
10 29
21 26
