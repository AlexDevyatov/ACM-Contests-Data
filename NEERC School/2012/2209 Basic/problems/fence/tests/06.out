8
-29 -10
-26 -26
1 -29
5 -26
27 4
28 26
21 26
10 29
